lib/p2p/Pool.ts

```diff
@@ -54,6 +54,13 @@ class Pool extends EventEmitter {
       return;
     }
 
+    // TODO differentiate between testnet & mainnet
+    this.logger.info('Connecting to Seed Peers');
+    this.addOutbound(new SocketAddress('xud1.test.exchangeunion.com', 8885));
+    this.addOutbound(new SocketAddress('xud2.test.exchangeunion.com', 8885));
+    this.addOutbound(new SocketAddress('xud3.test.exchangeunion.com', 8885));
+
+    this.logger.info('Connecting to Known / Previously Connected Peers');
```

Comment:
For the log statements, I don't think we should capitalize any words beyond the first one unless it's the proper name of something like a class.